setup.py

```diff
@@ -41,31 +41,23 @@ def run(self):
     namespace_packages=['chevah'],
     packages=find_packages('.'),
     package_data={
-        'chevah.github_hooks_server': ['static/*', 'author-aliases.txt'],
+        'chevah.github_hooks_server': ['author-aliases.txt'],
         },
-    scripts=['scripts/start-chevah-github-hooks.py'],
     install_requires=[
-        'klein==17.2',
         'toml',
-        # We keep an older version of python is use.
-        'Twisted==15.5.0.chevah1',
-        'github3-py==1.0.0.gitc82e90e',
-        # We don't depended directly on them but github3.py
-        # dependencies are bad.
-        'urllib3',
-        'chardet',
-        'certifi',
-        'idna',
+        'github3.py',
```

Comment:
I did not tesst the actual GitHub integration. It needs the secret, and to be deploymed.

Reply:
We can test it in a separate PR... it's ok to reduce the scope of this PR and just focus and nodejs tooling and deployment

we can fix the actual GH code in a separate PR.

the tests will need refactoring 

the private file is where you store a github API token... i think, it was a local file to prevent leaking the code via the env or process arguments.

there are some end-to-end tests and they are more suited for manual usage and not automated tests